pkg/controller/daemon/daemon_controller.go

```diff
@@ -801,6 +801,68 @@ func (dsc *DaemonSetsController) resolveControllerRef(namespace string, controll
 	return ds
 }
 
+// syncPodOnNode sync the DaemonSet Pods on the node, the return values are:
```

Comment:
s/sync/syncs/

The function returns four values. Only two are described in the comment.

Reply:
done